clsp/utils.clib

```diff
@@ -60,14 +60,14 @@
     )  
   )
 
-  ;;;(defun-inline calculate_nft_ownership_layer_puzzle_hash (NFT_OWNERSHIP_LAYER_MOD_HASH new_owner transfer_program_hash nft_inner_puzzle_hash)
-  ;;;    (curry_hashes NFT_OWNERSHIP_LAYER_MOD_HASH
-  ;;;      (sha256 1 nft_ownership_layer_mod_hash)
-  ;;;      (sha256 1 new_owner)
-  ;;;      transfer_program_hash
-  ;;;      inner_puzzle_hash
-  ;;;    )
-  ;;;)
+  (defun-inline calculate_nft_ownership_layer_puzzle_hash (NFT_OWNERSHIP_LAYER_MOD_HASH current_owner transfer_program_hash inner_puzzle_hash)
+      (curry_hashes NFT_OWNERSHIP_LAYER_MOD_HASH
+        (sha256 1 NFT_OWNERSHIP_LAYER_MOD_HASH)
+        (sha256 1 current_owner)
```

Comment:
Thanks. I like the current_owner terminology better. I was trying to base it on what the ownership layer called it, but I agree it didn't make sense.

Reply:
When I look at `nft_ownership_layer.clsp` in the chia-blockchain repo, they call that parameter CURRENT_OWNER too

Reply:
Which branch are you on? I see new_owner a bunch of places.